src/Apps/Artwork/Components/__tests__/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { ArtworkWithEstimateAndPremium } from "Apps/__tests__/Fixtures/Artwork/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo"
+import { ArtworkSidebarAuctionPartnerInfoFragmentContainer } from "Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarAuctionPartnerInfo"
+import { renderRelayTree } from "DevTools"
+import { graphql } from "react-relay"
+
+jest.unmock("react-relay")
+
+describe("ArtworkSidebarAuctionPartnerInfo", () => {
+  const getWrapper = async response => {
+    return await renderRelayTree({
+      Component: ArtworkSidebarAuctionPartnerInfoFragmentContainer,
+      query: graphql`
+        query ArtworkSidebarAuctionPartnerInfo_Test_Query {
+          artwork(id: "auction_artwork_estimate_premium") {
+            ...ArtworkSidebarAuctionPartnerInfo_artwork
+          }
+        }
+      `,
+      mockResolvers: {
+        Artwork: () => response,
+      },
+    })
+  }
+
+  let artwork
+
+  describe("ArtworkSidebarAuctionPartnerInfo", () => {
+    beforeEach(() => {
+      artwork = Object.assign({}, ArtworkWithEstimateAndPremium)
+    })
+
+    it("displays partner name, estimate and premium", async () => {
+      const wrapper = await getWrapper(artwork)
+
+      expect(wrapper.text()).toContain(artwork.partner.name)
```

Comment:
I would avoid writing tests that are bound to fixtures -- its slightly safer (and easier to grok) working with plain strings.

Reply:
The string is coming from a fixture:) But will happily update!